server/src/main/java/org/elasticsearch/index/mapper/NumberFieldMapper.java

```diff
@@ -126,12 +127,16 @@ public void createFields(Consumer<IndexableField> onField,
                                      boolean indexed,
                                      boolean docValued,
                                      boolean stored) {
-                if (indexed) {
-                    onField.accept(new FloatPoint(name, value.floatValue()));
-                }
-                if (docValued) {
-                    onField.accept(new SortedNumericDocValuesField(name,
-                        NumericUtils.floatToSortableInt(value.floatValue())));
+                if (indexed && docValued) {
+                    onField.accept(new FloatField(name, value.floatValue(), Store.NO));
```

Comment:
Required to pass https://github.com/crate/crate/blob/b84fd0ffbbc3570b08b91e3adb07f6c4e0ccf9b3/server/src/testFixtures/java/io/crate/types/DataTypeTestCase.java#L133

`ValueIndexer` and DocMapper parser must result in the same amount of fields